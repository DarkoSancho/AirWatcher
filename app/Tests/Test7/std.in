government official
6
50
2019-01-01 00:00:00
2019-01-03 23:59:59
0
40
0
40