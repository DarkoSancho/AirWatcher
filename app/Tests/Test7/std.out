==== RESULTS ====

Sensor: Sensor50  
Location: Latitude 46.0, Longitude -1.0  
Period: from 2019-01-01 00:00:00 to 2019-01-03 23:59:59

Total number of measurements: 12

Average [O3]   = 63.0 µg/m³  
Min [O3]       = 62.2 µg/m³  → 03/01/2019 12:00:00  
Max [O3]       = 63.7 µg/m³  → 02/01/2019 12:00:00  
Standard deviation [O3] = 0.8 µg/m³  

Average [NO2]   = 53.7 µg/m³  
Min [NO2]       = 52.4 µg/m³  → 03/01/2019 12:00:00  
Max [NO2]       = 55.0 µg/m³  → 02/01/2019 12:00:00  
Standard deviation [NO2] = 1.3 µg/m³  

Average [SO2]   = 49.0 µg/m³  
Min [SO2]       = 48.3 µg/m³  → 01/01/2019 12:00:00  
Max [SO2]       = 50.0 µg/m³  → 02/01/2019 12:00:00  
Standard deviation [SO2] = 0.9 µg/m³  

Average [PM10]   = 45.2 µg/m³  
Min [PM10]       = 44.6 µg/m³  → 01/01/2019 12:00:00  
Max [PM10]       = 45.6 µg/m³  → 03/01/2019 12:00:00  
Standard deviation [PM10] = 0.6 µg/m³  

Exceedances [PM10 > 40 µg/m³] : 3 times  
Exceedances [NO2 > 40 µg/m³]  : 3 times

ATMO Sub-indices based on average values:
- O3 = 63.0 µg/m³   → ATMO index: 4  
- NO2 = 53.7 µg/m³   → ATMO index: 3  
- SO2 = 49.0 µg/m³   → ATMO index: 3  
- PM10 = 45.2 µg/m³   → ATMO index: 3
